ext/sqlite3/statement.c

```diff
@@ -364,6 +364,22 @@ column_count(VALUE self)
     return INT2NUM(sqlite3_column_count(ctx->st));
 }
 
+#if HAVE_RB_ENC_INTERNED_STR_CSTR
+static VALUE
+interned_utf8_cstr(const char * str)
+{
+    return rb_enc_interned_str_cstr(str, rb_utf8_encoding());
+}
+#else
+static VALUE
+interned_utf8_cstr(const char * str)
+{
+    VALUE rb_str = rb_utf8_str_new_cstr(str);
+    rb_obj_freeze(rb_str);
```

Comment:
Minor: this call is unnecessary, because `String#-@` always returns a frozen String.

Reply:
Thanks, fixed.